bench/nbson_bench.erl

```diff
@@ -111,8 +111,8 @@ bench_encode(Path, Times) ->
     io:format("~20.. s  ~20.. B  ~20.. B  ~20.. B~n",
               [DocCount,
                byte_size(Bin),
-               round(NbsonTimeEncode/Times),
-               round(BsonErlangTimeEncode/Times)]).
+               NbsonTimeEncode,
+               BsonErlangTimeEncode]).
```

Comment:
The benchmark results in the README.md file should be updated.

Reply:
Updated